.gitignore

```diff
@@ -11,4 +11,5 @@ docs/.yarn/
 !.yarn/versions
 .next
 .idea
-.vscode
\ No newline at end of file
+.vscode
+*.txt
```

Comment:
This is not the way, better to ignore the file that is know the test to output to.